bin/setup

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# This assumes you have general prerequisites installed as by:
+# https://github.com/artsy/potential/blob/master/scripts/setup
+
+# Exit if any subcommand fails
+set -e
+
+echo "brew update and bundle install..."
+brew update
+brew bundle --file=- <<EOF
+brew asdf
```

Comment:
We are on elixir `1.10.3` which I cannot install via `brew`. So relying on `asdf`. Installing`asdf` here rather than in Potential's [setup script](https://github.com/artsy/potential/blob/main/scripts/setup), since I am not aware of any other project that can benefit from it.